core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5669,6 +5669,9 @@ components:
           type: boolean
           description: Optional, false if not specified. If true, the workspace is created with a Billing Project owner but no workspace owner. Requires being a Billing Project owner.
           default: false
+        bucketLocation:
+          type: string
+          description: Region in which bucket attached to the workspace should be created. It is optional. If not provided, the bucket will be created in `us-central1`
```

Comment:
```suggestion
          description: Region (NOT multi-region) in which bucket attached to the workspace should be created. If not provided, the bucket will be created in the 'US' multi-region.
```

Reply:
You can get the "It is optional" by adding a `required: false` field under `bucketLocation` in the yaml.